tsup.config.ts

```diff
@@ -55,13 +56,14 @@ const browserConfig: Options = {
   name: 'browser',
   platform: 'browser',
   entry: ['./src/browser/index.ts'],
-  external: [mswCore, ecosystemDependencies],
+  external: [ecosystemDependencies],
   format: ['esm', 'cjs'],
   outDir: './lib/browser',
   bundle: true,
   splitting: false,
   sourcemap: true,
   dts: true,
+  noExternal: Object.keys(packageJson.dependencies),
```

Comment:
This introduces an impossible state. `noExternal`, I imagine, takes precedence over `external`, and so we end up bundling ecosystem dependencies. Is that the case?